### DISCUSSION TITLE
Where can I see billing info for GitHub Actions?

### DISCUSSION THREAD

**Discussion by ryanvanderpol** [community/community#27137] [Actions]
<p>I know that the first 2,000 minutes are free (for private repos) and after that it’s billed by the minute, per OS. Presumably, I will be billed for usage across all repos in my organization at the end of the month, but I can’t find anywhere in my account where I can see this.</p>
<p>So, where can I see:</p>
<ol>
<li>How many minutes I’ve currently used across all repos my organization?</li>
<li>A live total of my bill for the month?</li>
</ol>

**Comment by bendavies** [comment]
<p>+1.</p>
<p>I’ve also emailed GitHub support about this and have received no reply. </p>

**Comment by wipe2238** [comment]
<p>See <a href="https://github.community/t/total-amount-of-minutes-used/16162/2">https://github.community/t5/GitHub-Actions/Total-amount-of-minutes-used/m-p/30690/highlight/true#M514</a></p>

**Comment by chrispat** [comment, marked as answer]
<p>We are still working on enabling billing as part of our work to make GitHub Actions GA. Prior to us turning billing on you will start to see a $0 charge and a meter for consumed minutes showing up in your account settings.</p>

**Comment by chrispat** [comment]
<p>You should soon see the ability to export a CSV of your billing details in your account or Org.  That export will enable you to report on the minutes used by specific repos and other dimenions.</p>

**Comment by dgancho** [comment]
<p>Hello, <a class="mention" href="/chrispat">@chrispat</a> Are there any updates here? How to see the number of minutes used by GitHub Actions for an org?</p>

**Comment by stefanbuck** [comment]
<p>The CSV export is great and provides a detail breakdown of all costs. However, I wonder if an API endpoint is planed to make those data accessible to other systems <a class="mention" href="/chrispat">@chrispat</a>?</p>
